script/build

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
How does `script/build` fit into the normal process of things? At first glance I expected `script/test` might call `script/build`, but it calls `script/compile` instead.